### ISSUE TITLE
Show date as well as time on the X axis

### ISSUE THREAD

**Issue by squaregoldfish** [danvk/dygraphs#842]
This is kind of the opposite issue to #369.

When you zoom into a graph with time as the X axis, the labels show only the time and not the date. If you're navigating around the plot it can be easy to forget exactly where you are, and it's hard to work it out if you're not close to midnight and/or there's no legend. It would be nice to somehow show the date along with the times.

This is something I could look into implementing in a few months' time if no-one else gets to it first.

**Comment by danvk** [comment]
I agree that at least one label should always include a date. You could work around this by writing a `axisLabelFormatter`, or a custom `ticker`.
